apimonitor/settings.py

```
@@ -0,0 +1,171 @@
+from __future__ import annotations
+
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)